src/modules/LibraryCard/components/Login.tsx

```diff
@@ -38,20 +38,22 @@ export const Login = ({saveDetails}: IProps) => {
       setIsLoading(true);
 
       const response = await fetch(
-        'https://kyyti.koha-suomi.fi/api/v1/auth/session',
+        'https://kyyti.koha-suomi.fi/api/v1/borrowers/status',
         {
           method: 'POST',
           headers: {
             'Content-Type': 'application/x-www-form-urlencoded',
             Accept: 'application/json',
           },
-          body: `cardnumber=${inputCardNumber}&password=${password}`,
+          body: `uname=${inputCardNumber}&passwd=${password}`,
```

Comment:
The value of `uname` is unclear here. Maybe @villemettala could clarify.

Reply:
current naming is that cardnumber --> uname , so this is correct